<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>899</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidgetTree">
      <property name="currentIndex">
       <number>11</number>
      </property>
      <widget class="QWidget" name="tabMySQL">
       <attribute name="title">
        <string>MySQL</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="4">
         <widget class="QPushButton" name="btnMySqlReconn">
          <property name="text">
           <string>重连</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="btnMySQLConn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>连接</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="5">
         <widget class="QTextEdit" name="textEditMySQLBuffer">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="5">
         <widget class="QTextEdit" name="textEditMySQLLog"/>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="btnMySQLQuery">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>执行</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabTableView">
       <attribute name="title">
        <string>TableView</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="1" column="0" colspan="5">
         <widget class="QTableView" name="tableView">
          <property name="cornerButtonEnabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="5">
         <widget class="QTableView" name="tableView_2"/>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="btnTableViewInit">
          <property name="text">
           <string>Init</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QPushButton" name="btnTableViewSort">
          <property name="text">
           <string>排序</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="btnTableViewGridStyle">
          <property name="text">
           <string>网格线</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="btnTableViewAdd">
          <property name="text">
           <string>插入</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboBox"/>
        </item>
       </layout>
      </widget>
      <widget class="CWidgetTreeView" name="tabTreeView">
       <attribute name="title">
        <string>TreeView</string>
       </attribute>
      </widget>
      <widget class="CWidgetWeb" name="tabWeb">
       <attribute name="title">
        <string>Web</string>
       </attribute>
      </widget>
      <widget class="CWidgetMedia" name="tabMedia">
       <attribute name="title">
        <string>播放器</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabTCP">
       <attribute name="title">
        <string>TCP</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="CWidgetTCPServer" name="tab_3">
           <attribute name="title">
            <string>服务器</string>
           </attribute>
          </widget>
          <widget class="CWidgetTCPClient" name="tab_4">
           <attribute name="title">
            <string>客户端</string>
           </attribute>
          </widget>
          <widget class="CWidgetTCPClient" name="tab_5">
           <attribute name="title">
            <string>客户端1</string>
           </attribute>
          </widget>
          <widget class="CWidgetTCPClient" name="tab_6">
           <attribute name="title">
            <string>客户端2</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="CWidgetMenu" name="tabMenu">
       <attribute name="title">
        <string>右键菜单</string>
       </attribute>
      </widget>
      <widget class="CWidgetProblem" name="tabSuanFa">
       <attribute name="title">
        <string>算法</string>
       </attribute>
      </widget>
      <widget class="CWidgetMap" name="tabMap">
       <attribute name="title">
        <string>地图</string>
       </attribute>
      </widget>
      <widget class="CWidgetHttp" name="tabHttp">
       <attribute name="title">
        <string>Http下载</string>
       </attribute>
      </widget>
      <widget class="CWidgetEvent" name="tabEvent">
       <attribute name="title">
        <string>事件</string>
       </attribute>
      </widget>
      <widget class="CWidgetDrop" name="tabDrop">
       <attribute name="title">
        <string>文件拖放</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>899</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CWidgetWeb</class>
   <extends>QWidget</extends>
   <header>widgetweb.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CWidgetMedia</class>
   <extends>QWidget</extends>
   <header>widgetmedia.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CWidgetTCPServer</class>
   <extends>QWidget</extends>
   <header>widgettcpserver.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CWidgetTCPClient</class>
   <extends>QWidget</extends>
   <header>widgettcpclient.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CWidgetMenu</class>
   <extends>QWidget</extends>
   <header>widgetmenu.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CWidgetProblem</class>
   <extends>QWidget</extends>
   <header>widgetproblem.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CWidgetMap</class>
   <extends>QWidget</extends>
   <header>widgetmap.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CWidgetTreeView</class>
   <extends>QWidget</extends>
   <header>widgettreeview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CWidgetHttp</class>
   <extends>QWidget</extends>
   <header>widgethttp.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CWidgetEvent</class>
   <extends>QWidget</extends>
   <header>widgetevent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CWidgetDrop</class>
   <extends>QWidget</extends>
   <header>widgetdrop.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
